framework/src/st_things/things_stack/utils/things_network.h

```diff
@@ -43,4 +43,8 @@ void things_tcp_session_state_cb(const CAEndpoint_t *info, bool connected);
 
 int things_set_ap_connection(access_point_info_s *ap_info);
 
+int things_wifi_scan_ap();
```

Comment:
please add void in parameter for build waring.

Reply:
@JeongJunSik  fixed.